test/sharness/Makefile

```diff
@@ -16,6 +16,7 @@ all: clean deps $(T) aggregate
 clean:
 	@echo "*** $@ ***"
 	-rm -rf test-results
+	-rm -rf bin/ipfs
```

Comment:
I would make changes and re-run tests, but tests would be executed against the old binary. What'd the preferred way to have the binary rebuilt?